### DISCUSSION TITLE
[A-L] (14 Luglio 2023) Esercizio 1

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#392] [Esami A-L]
Il menu di un ristorante comprende 5 antipasti (di tre tipi: 2 di carne, 1 di pesce, 2 vegetariani), 8 primi (di tre tipi: 4 di carne, 2 di pesce e 2 vegetariani) e 7 secondi (di tre tipi: 3 di carne, 2 di pesce e 2 vegetariani).

1. In quanti modi posso scegliere un pasto completo (antipasto, primo e secondo)? [1 punto]

2. In quanti modi posso scegliere un pasto completo senza piatti di carne? [2 punti]

3. Mario non mangia carne e Maria non mangia pesce. Quante sono le possibili ordinazioni di due pasti completi fatte da Mario e Maria? 
[2 punti]

**Comment by ZacPool** [comment]
![357526498-5c0698ed-f07b-431e-9e86-3116967ca523](https://github.com/user-attachments/assets/cfe0f91e-e39e-4c43-9b0a-900b28536543)

Ditemi se giusto pls 🙏🏻

**Reply by FeddyLix17** [reply]
tutto giusto tranne il simbolo delle combinazioni dove la sintassi è $\Large \binom{n}{k}$ (dove n è il numero di oggetti totali e k il numero di oggetti che uno sceglie) invece di $\Large \binom{k}{n}$

**Comment by FeddyLix17** [comment]
Definiti A, P e S rispettivamente come gli insiemi degli antipasti, primi e secondi

1. [principio moltiplicativo](https://www.mathcamp.it/lezioni.php?less=principio_di_moltiplicazione)

$$
    \Large |\ \texttt{Pasto completo}| =  |A| * |P| * |S| = 5 * 8 * 7 = 280
$$

---

2. come il punto 1 escludendo da ogni insieme i tipi di piatti di carne ( definiamo questi insiemi rispettivamente come $\large A{nc},\ P{nc},\ S{nc}$ )

$$
    \Large |\ \texttt{Pasto completo senza carne}| =  |A{nc}| * |P{nc}| * |S{nc}| = 3 * 4 * 4 = 48
$$

---

3. definiamo $\large A{np},\ P{np},\ S{np}$ come sottoinsiemi rispettivamente di A, P e S in cui non sono presenti piatti di pesce

$$
    \Large |\ \texttt{Pasto completo senza pesce}| =  |A{np}| * |P{np}| * |S{np}| = 4 * 6 * 5 = 120
$$

da cui

$$
    \Large 
    \begin{align}
    |\ \texttt{ordinazioni due pasti completi Mario e Maria}| = \\
    = |\ \texttt{Pasto completo senza carne}| *  |\ \texttt{Pasto completo senza pesce}| =\\
    = 48 * 120 = 5760
     \end{align}
$$
